### PR TITLE
Create a `data.js` util file.

### PR DESCRIPTION
**Pull request by Tazaf**
It should contains utility functions related to data(base) manipulation.
Right now, only the `create` function is present. It's used in the
`sample-data.js` script.

This function ought to be used in the mobile e2e tests also,
for creating fake data in the test database.

(Also, package lock update)

### CODE REVIEW

**Comment by AlphaHydrae** [comment]
I'm not entirely sure why those changes are here in the first place. I would never except an `npm install` command to **downgrade** packages.

However, I just did some reading and it appears the behaviour of `npm install` was modified recently (within the past year). It now seems that `npm install` will update dependencies despite the presence of a `package-lock.json`, if new versions matching the ranges in `package.json` are found (e.g. with `~1.0.0` in `package.json` and `1.0.0` locked in `package-lock.json`, an `npm install` command will update `package-lock.json` if a new `1.1.0` version is available, for example).

A new `npm ci` command was introduced to install purely based on the `package-lock.json` file and not make any changes whatsoever (`npm help ci` says "Install a project with a clean slate [...] It will never write to package.json or any of the package-locks: installs are essentially frozen.").

Basically we have to re-learn using `npm`: we should use `npm ci` to make a clean install instead of `npm install`. What is not clear to me yet is the difference between `npm install` and `npm update` with this new behaviour added into the mix...

* https://stackoverflow.com/questions/45022048/why-does-npm-install-rewrite-package-lock-json
* https://github.com/npm/npm/issues/17979#issuecomment-332701215

**Comment by Tazaf** [comment]
Well... that was not exactly what I wanted to do.

Let's start again.
I'll do a single commit and force push it to this PR.
